mage/query-modules/python/elasticsearch-synchronization.md

```diff
@@ -0,0 +1,315 @@
+{label: 'Step 3: Create edge index', value: 'create-edge-index'},
+{label: 'Step 4: Populate db', value: 'insert'},
+{label: 'Step 5: Index database', value: 'index-database'},
+{label: 'Step 6: Scan', value: 'scan'},
+{label: 'Step 7: Scan results', value: 'scan-results'},
+{label: 'Step 8: Create trigger', value: 'trigger'},
+{label: 'Step 9: Insert node and edge', value: 'insert-new'},
+{label: 'Step 10: Search', value: 'search'},
+{label: 'Step 11: Search results', value: 'search-results'},
+]
+}>
+
+
+  <TabItem value="connect">
+
+```cypher
+CALL elastic_search_serialization.connect("https://localhost:9200", "http_ca.crt", "<ELASTIC_USER>","<ELASTIC_PASSWORD>") YIELD *;
+```
+
+  </TabItem>
+
+
+  <TabItem value="create-node-index">
+
+```cypher
+CALL elastic_search_serialization.create_index("docs_nodes", "node_index_path_schema.json",
+{analyzer: "mem_analyzer", index_type: "vertex"}) YIELD *;
+```
+
+  </TabItem>
+
+  <TabItem value="create-edge-index">
+
+```cypher
+CALL elastic_search_serialization.create_index("docs_edges",  "edge_index_path_schema.json", {analyzer: "mem_analyzer", index_type: "edge"}) YIELD *;
+```
+
+  </TabItem>
+
+
+  <TabItem value="insert">
+
+```cypher
+CREATE (n0 {name: "n0"}), (n1 {name: "n1"}), (n2 {name: "n2"}), (n3 {name: "n3"}), (n4 {name: "n4"}), (n5 {name: "n5"}), (n6 {name: "n6"});
+CREATE (n1)-[r1:RELATED]->(n2);
+CREATE (n1)-[r2:RELATED]->(n3);
+CREATE (n1)-[r3:RELATED]->(n3);
+CREATE (n2)-[r4:RELATED]->(n1);
+CREATE (n2)-[r5:RELATED]->(n5);
+CREATE (n2)-[r6:RELATED]->(n3);
+CREATE (n3)-[r7:RELATED]->(n6);
+CREATE (n3)-[r8:RELATED]->(n1);
+CREATE (n1)-[r9:RELATED]->(n4);
+```
+
+  </TabItem>
+
+  <TabItem value="index-database">
+
+```cypher
+CALL elastic_search_serialization.index_db("docs_nodes", "docs_edges", 4) YIELD *;
+```
+
+  </TabItem>
+
+  <TabItem value="scan">
+
+```cypher
+CALL elastic_search_serialization.scan("docs_nodes", "{\"query\": {\"match_all\": {}}}", "5m", false, false, 100, 0, 2.0, False) YIELD *;
+```
+
+  </TabItem>
+
+  <TabItem value="scan-results">
+
+  <img src={require('../../data/query-modules/python/elasticsearch-synchronization/scan_results.png').default}/>
+
+  </TabItem>
+
+  <TabItem value="trigger">
+
+```cypher
+CREATE TRIGGER elastic_search_create
+ON CREATE AFTER COMMIT EXECUTE
+CALL elastic_search_serialization.index(createdObjects, "docs_nodes", "docs_edges") YIELD * RETURN *;
+```
+
+  </TabItem>
+
+  <TabItem value="insert-new">
+
+```cypher
+CREATE (n7 {name: "n7"});
+MATCH (n6 {name: "n6"}), (n7 {name: "n7"})
+CREATE (n6)-[:NEW_CONNECTION {edge_property: "docs"}]->(n7);
+```
+
+  </TabItem>
+
+  <TabItem value="search">
+
+```cypher
+CALL elastic_search_serialization.search("docs_nodes",  "{\"match_all\": {}}", 1000, 0) YIELD *;
+```
+
+  </TabItem>
+
+  <TabItem value="search-results">
+
+  <img src={require('../../data/query-modules/python/elasticsearch-synchronization/trigger_results.png').default}/>
+
+  </TabItem>
+
+</Tabs>
```

Comment:
Unfortunately this is not very user-friendly due to the number of steps :/ I think the better course of action would be to use headlines

Reply:
I merged few steps, so I think it is better now :)